server/neptune/gateway/event/push_event_handler.go

```diff
@@ -123,6 +123,10 @@ func (p *PushHandler) startWorkflow(ctx context.Context, event Push, rootCfg *va
 				Credentials: workflows.AppCredentials{
 					InstallationToken: event.InstallationToken,
 				},
+				HeadCommit: workflows.HeadCommit{
+					// we only support branch refs
+					Ref: fmt.Sprintf("refs/heads/%s", event.Ref.Name),
```

Comment:
This is the format that is required for the clone I'm assuming?

Can we just pass an object similar to https://github.com/lyft/atlantis/blob/25d3a061e24f5658a70889d9eed73d1cb81bbc6b/server/vcs/ref.go#L13

and then just build that string on the fly as part of a receiver function on that object whenever we need it? it's a bit more flexible that way.